server/routes/routes.js

```diff
@@ -0,0 +1,100 @@
+const Product = require('Product model goes here');
+const User = require('User model goes here');
+const express = require('express');
+const router = express.Router();
+
+router.get('/', async (req, res, next) => {
```

Comment:
currently this is too general with a route called '/api'. change this later to  something like '/api/sessions'